app/api/enrolments.py

```diff
@@ -71,7 +103,19 @@ def get_callbacks_list_for_enrolment(
 @router.get("/enrolments/{enrolment_id}/journal/{callback_id}")
 def get_callback_for_enrolment(enrolment_id: str, callback_id):
     """
-    Returns callback details for an callback of an enrolment
+    <p>
+    Aged Care Provider (employer) can view callback details for an enrolment\n
+    He will get,
+    <ol>
+      <li><b>callback_id</b> identifier for enrolment</li>
+      <li><b>enrolment_id</b> identifier for callback</li>
+      <li><b>shared_secret</b> for verification</li>
+      <li><b>received</b> date and time when the callback was received</li>
+      <li><b>tp_sequence</b> sequence number from training provider source-system.
+      These are used for sorting in the order of sender’s intent.</li>
+      <li><b>payload</b> required</li>
```

Comment:
Callback's structure has changed. There is no tp_sequence anymore. Can you check it from the `employer-callback` repository?

Reply:
Checked it. I'll open a new PR for it. As it will change 3 usecases.